### PR TITLE
chore: add method to encode profile

### PR DESCRIPTION
**Pull request by nolanmar511**

### CODE REVIEW

File: ts/src/profile-encoder.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as pify from 'pify';
+import {gzip as gzipPromise, gzipSync} from 'zlib';
```
Comment (ofrobots) [inline ts/src/profile-encoder.ts]: `gzipPromise` is a misnomer. I would suggest importing as `gzip` here and using `gzipPromise` as a variable below.

**Comment by nolanmar511** [comment]
Kokoro system test is not implemented yet (it's a WIP; #12), so it is expected to fail.

All other tests are passing.

**Comment by nolanmar511** [comment]
I've addressed comments, but this change now should follow https://github.com/google/pprof-nodejs/pull/14.

File: ts/src/profile-encoder.ts (hunk 1 above)
Comment (nolanmar511) [reply ts/src/profile-encoder.ts]: Thanks! And done!

**Pull request merged**